tests/schema/test.locale-messagesjson.js

```diff
@@ -20,12 +20,11 @@ describe('messages', () => {
     expect(validateLocaleMessages.errors[0].dataPath).toEqual('/foo/message');
   });
 
-  it('should fail on invalid message name', () => {
+  it('should not fail on invalid message name', () => {
```

Comment:
How about changing this test description to something like: "should not validate the message name"?
it may be a bit confusing to call "invalid" something that doesn't make the validation to fail.

We should also add an empty message name now that the regular expression allows it.

Reply:
Also, coveralls still complains about this line: https://coveralls.io/jobs/34824362/source_files/652389568#L47 (thanks @EnTeQuAk for finding it, it was driving me a bit crazy :-))

If there is no way anymore for a message name to be invalid, it is probably reasonable to remove the branch and the related constant?

@EnTeQuAk How that sounds to you?